src/PayloadParser.h

```diff
@@ -316,9 +316,10 @@ namespace snowcrash {
             SectionType sectionType = pd.sectionContext();
 
             if (sectionType == RequestSectionType || sectionType == RequestBodySectionType) {
+
                 checkRequest(node, pd, out);
-            }
-            else if (sectionType == ResponseSectionType || sectionType == ResponseBodySectionType) {
+            } else if (sectionType == ResponseSectionType || sectionType == ResponseBodySectionType) {
```

Comment:
This happened because of revert. I have been slowly converting the `} else if` to `else if`.